crates/holochain/src/bin/holochain/main.rs

```diff
@@ -124,18 +124,24 @@ async fn conductor_handle_from_config_path(
         load_config(&config_path, config_path_default)
     };
 
-    // If interactive mode, give the user a chance to create LMDB env if missing
+    // Check if LMDB env dir is present
+    // In interactive mode give the user a chance to create it, otherwise create it automatically
     let env_path = PathBuf::from(config.environment_path.clone());
-    if interactive && !env_path.is_dir() {
-        match interactive::prompt_for_environment_dir(&env_path) {
-            Ok(true) => println!("LMDB environment created."),
-            Ok(false) => {
-                println!("Cannot continue without LMDB environment set.");
-                std::process::exit(ERROR_CODE);
-            }
-            result => {
-                result.expect("Couldn't auto-create environment dir");
+    if !env_path.is_dir() {
+        if interactive {
+            match interactive::prompt_for_environment_dir(&env_path) {
+                Ok(true) => println!("LMDB environment created."),
+                Ok(false) => {
+                    println!("Cannot continue without LMDB environment set.");
+                    std::process::exit(ERROR_CODE);
+                }
+                result => {
+                    result.expect("Couldn't auto-create LMDB environment dir");
```

Comment:
I know this was in the original code, but now that we are touching it, I think we shouldn't panic here, but exit more cleanly with an error code.  The case to test against is if you don't have permissions to the directory provided.